plugin/contractpb/contractpb.go

```diff
@@ -43,8 +43,8 @@ type Context interface {
 	plugin.VolatileAPI
 	StaticContext
 	Set(key []byte, pb proto.Message) error
-	Delete(key []byte)
-	HasPermission(token []byte, roles []string) (bool, []string)
+	Delete(key []byte) error
+	HasPermission(token []byte, roles []string) (bool, []string, error)
```

Comment:
did you update loomchain? why do we need an error returned here

Reply:
loomchain doesn't seem to compile

Reply:
Have fixed the same.